docs/contents/dev-docs/building-controller.md

```diff
@@ -0,0 +1,83 @@
+# Building an ACK service controller
+
+This document continues our contributor-focused discussion by explaining how to
+build or regenerate an ACK service controller.
+
+## Prerequisites
+
+You should have forked the `github.com/aws-controllers-k8s/code-generator`
+repository and `git clone`'d it locally when [setting up](../setup) your
+development environment,
+
+With the prerequisites out of the way, let's move on to the first step:
+building the code generator.
+
+## Build code generator
+
+Building an ACK service controller (or regenerating an existing one from a
+newer API model file) requires the `ack-generate` binary, which is the main
+code generator CLI tool.
+
+To build the latest `ack-generate` binary, execute the following command from
+the root directory of the `github.com/aws-controllers-k8s/code-generator`
+source repository:
+
+```
+make build-ack-generate
```

Comment:
This did not work for me from the new code-generator repo and I had to run `go get -u -tags codegen github.com/aws-controllers-k8s/code-generator/cmd/ack-generate`
Could be that I am missing something.
Am I missing something ?

Reply:
hmm, @mbaijal not sure... perhaps you need to pull the latest from the code-generator repo? The Makefile target for build-ack-generate is in there:

https://github.com/aws-controllers-k8s/code-generator/blob/main/Makefile#L24-L27

Reply:
I remember checking out this branch to test it. For now, this should be ok to merge since it works for you. I will re-test post merge.